packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js

```diff
@@ -77,17 +82,17 @@ function CellTypeWheelExperimentHeatmap(props) {
         enableSpeciesSelect={true}
         speciesSelectClassName={`small-4 columns`}
         defaultSpecies={props.species}
+        onSpeciesSelectOnChange={handleChange}
       />
       <div className={`row-expanded small-12 columns`}>
         <div className={`small-12 medium-6 columns`} aria-label={`Cell type wheel`}>
           {props.searchTerm.trim() ?
             <CellTypeWheelFetchLoader
               host={props.host}
-              resource={URI(props.cellTypeWheelResource)
+resource={URI(props.cellTypeWheelResource)
```

Comment:
I know it is a tiny formatting thing, but could you restore the aligning of this line, please?

Reply:
sorry, not sure, how it was disturbed, it makes sense. Thanks for that.

Reply:
I fixed formatting, please feel free to review when log in